### DISCUSSION TITLE
ScriptReference/Mesh.MeshData

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#525] [Notes]
# ScriptReference/Mesh.MeshData

https://docs.unity3d.com/ScriptReference/Mesh.MeshData.html

**Comment by radiatoryang** [comment]
for a better code example that actually uses this in a Jobs / Burst context, see Aras-p's CreateOneMeshFromWholeScene.cs Mesh.MeshData example gist -- https://gist.github.com/aras-p/f0eeec6abcf9690b6b5fb83376076456
